<launch>
  <!-- Octomap server node -->
  <node pkg="octomap_server" type="octomap_server_node" name="octomap_server" output="screen">
    
    <!-- Fixed world frame -->
    <param name="frame_id" value="odom"/>

    <!-- Set resolution of the voxel grid -->
    <param name="resolution" value="0.025"/>

    <!-- Use depth image to build map -->
    <param name="sensor_model/max_range" value="4.0" />
    <param name="sensor_model/hit" value="0.7" />
    <param name="sensor_model/miss" value="0.4" />
    <param name="sensor_model/min" value="0.12" />
    <param name="sensor_model/max" value="0.97" />

    <!-- Topics to subscribe -->
    <remap from="cloud_in" to="/hsrb/head_rgbd_sensor/depth_registered/rectified_points"/>
  </node>
</launch>
